.eslintrc.json

```diff
@@ -1,3 +1,5 @@
 {
-  "extends": "@antfu"
+  "extends": "@antfu",
+  "plugins": ["eslint"],
+  "parser": "@typescript-eslint/parser"
```

Comment:
I don't think this is needed 🤔

Reply:
I should have noted where this issue came up but I was getting a linting error that I couldn't avoid without bringing in these changes. If need be I can just remove this at the end of the review if you're happy otherwise. Would that be ok?

Reply:
Are you using pnpm? I don't have this problem in my other setups

Reply:
yes I use **pnpm** religiously ... actually I don't believe I've every prayed to **pnpm** but I would if it would make this problem go away :)